client/src/boot/registerComponents.js

```diff
@@ -0,0 +1,20 @@
+import Injector from 'lib/Injector';
+import readOneBlockQuery from 'state/history/readOneBlockQuery';
+import revertToBlockVersionMutation from 'state/history/revertToBlockVersionMutation';
+
+
+export default () => {
+  Injector.transform(
+    'elements-history',
+    (updater) => {
+      // Add content block history to the HistoryViewer
+      updater.component(
+        'HistoryViewer.Form_ItemEditForm',
+        readOneBlockQuery,
+        'ElementHistoryViewer'
+      );
+      }
+  );
+
+
+};
```

Comment:
This file can be removed, since it was renamed in #181 to `registerTransforms.js`